packages/paste-website/src/pages/components/breadcrumb/index.mdx

```diff
@@ -0,0 +1,237 @@
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Breadcrumb, BreadcrumbItem} from '@twilio-paste/breadcrumb';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import {Truncate} from '@twilio-paste/truncate';
+import Changelog from '@twilio-paste/breadcrumb/CHANGELOG.md';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/breadcrumb"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/breadcrumb/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/breadcrumb"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-breadcrumb--anchors"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<Callout>
+  <CalloutTitle>Using alpha components in your product</CalloutTitle>
+  <CalloutText>
+    Alpha components are ready for use in production and components like Breadcrumb won't likely see any API changes.
+    The term "Alpha" means there is only basic documentation and no corresponding Sketch symbol at this time. Those will
+    eventually be prioritized, but please share your experience using this Alpha component by{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions/new">starting a discussion on GitHub</Anchor>
+    If you’re interested in helping to symbolize this component in Sketch, we’d love your help – please reach out!
+  </CalloutText>
+</Callout>
+
+## Guidelines
+
+### About Breadcrumb
+
+Breadcrumb navigation helps the user understand their location in an app or website by providing a trail back to the top level of the website.
+
+Breadcrumbs are listed in order and represent the page hierarchy. They will include links to parent pages and can optionally show the current page.
+
+### Accessibility
+
+- The Breadcrumb component includes the accessible `aria-label="breadcrumb"` attribute.
+- An unlinked BreadcrumbItem includes the accessible `aria-current="page"` attribute.
+
+<Callout variant="warning">
+  <CalloutText>
+    The unlinked `BreadcrumbItem` should <strong>ONLY</strong> be used for the page a user is currently on. If it is not
+    the current page, make sure to add the `href` so the `BreadcrumbItem` goes to the appropriate location.
```

Comment:
The backticks should be removed here, or the code style should be added

Reply:
Good catch. I added the inline code blocks.